draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -1151,20 +1164,21 @@ uCDN-A                         dCDN-B                         dCDN-C
 
 <section title="CI/T Object Properties and Encoding" anchor="cit-objects-properties-and-encoding">
    <t>
-           Trigger and trigger collection resources as well as their properties, are encoded using JSON, as defined in
-        <xref target="ci-trigger-resource"/>, and <xref target="ci-trigger-collection-resource"/>, respectively. 
-        When sending the JSON-based representation of these resources, the MIME media type 
-        "application/cdni" MUST be used, with parameter "ptype" values as defined below and 
-        in <xref target="cdni-payload-type"/>.
+   Trigger and trigger collection resources as well as their properties, are encoded using JSON, as defined in
```

Comment:
This could be reworded for better readability, maybe:
Triggers, trigger collections, and their properties are encoded using JSON, ..."